sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2304,7 +2304,15 @@ object functions {
    * @group string_funcs
    * @since 1.5.0
    */
-  def ltrim(e: Column): Column = withExpr {StringTrimLeft(e.expr) }
+  def ltrim(e: Column): Column = withExpr {StringTrimLeft(e.expr)}
```

Comment:
Hello Sean,  sorry, just want to clarify whether I should change the existing three trim function style or not. The above example is from existing trim function (@since 1.5.0). Should I leave them as it is? Thanks.